src/EditorFeatures/CSharpTest/QualifyMemberAccess/QualifyMemberAccessTests.cs

```diff
@@ -1028,5 +1028,57 @@ void Error()
 }",
 CodeStyleOptions.QualifyMethodAccess);
         }
+
+        [WorkItem(17711, "https://github.com/dotnet/roslyn/issues/17711")]
```

Comment:
❓ Can we add a test for fields here? It would be similar to the other tests except it wouldn't use the virtual/override combination.